src/plugins/intel_cpu/tests/functional/shared_tests_instances/behavior/plugin/preprocessing.cpp

```diff
@@ -37,5 +35,3 @@ INSTANTIATE_TEST_SUITE_P(BehaviourPreprocessingTestsViaGetBlob, PreprocessingPre
                                 ::testing::ValuesIn(configs)),
                         PreprocessingPrecisionConvertTest::getTestCaseName);
```

Comment:
I suppose `BehaviourPreprocessingTestsViaGetBlob` are old tests? and we can remove them?

Reply:
Test is removed

Reply:
I reverted deletion to unblock merge because openvino_contrib and npu tests depend on it